### ISSUE TITLE
12 instead of 42

### ISSUE THREAD

**Issue by PlumeRoberts** [42-AI/bootcamp_python#254]
* Day: 02
* Exercise: 01

```py
obj = what_are_the_vars(42, "Yes", a=10, var_2="world")
```
outputs
```
a: 10
var_0: 12
var_1: Yes
var_2: world
end
```
but `var_0` should be `42`
## Fixed on:
- [ ] Github
- [ ] Gitlab

**Comment by maperez42** [comment]
Fixed in coming 3.2

**Comment by maperez42** [comment]
Fixed in 3.2
